rpm/qtdeclarative.spec

```diff
@@ -268,6 +285,14 @@ touch .git
 
 make %{?_smp_mflags}
 
+%if 0%{?qt5_release_build}
```

Comment:
I'm not totally sure where it comes from. I think it may be set by the prjconf in OBS. I'm not totally sure, @iamer would have a better answer than me (and is probably also a good candidate to sanity-check this) :)

I would guess that:

%define qt5_release_build 1

might help to test it.